config/development.json

```diff
@@ -133,6 +133,7 @@
 		"page/export": true,
 		"pattern-assembler/client-side-render": true,
 		"pattern-assembler/logged-out-showcase": true,
+		"pattern-assembler/sidebar-revamp": false,
```

Comment:
Are we planning to turn this feature flag in development after this PR's been approved?

Reply:
I've turned it as we will keep using the current design